src/lustre/lustreAst.ml

```diff
@@ -17,11 +17,9 @@
 *)
 
 open Lib
-          
+
 exception Parser_error
 
-module SI = Set.Make (Ident)
-          
 let error_at_position pos msg =
```

Comment:
You already moved `error_at_position` and `warn_at_position` to `lustreAstHelpers.ml`. You should remove them from `lustreAst.ml`

Reply:
fixed this now.